backend/tournesol/resources/export_readme.txt

```diff
@@ -18,43 +18,116 @@ List of files
+  user can choose to give to another account if they trust that the user
+  controlling the account is authentic and controls a single account. Trust
+  scores influence the computation of voting rights (see file
+  individual_criteria_scores.csv).
 
 individual_criteria_scores.csv
 ------------------------------
 
-Type: algorithm output.
-
 This file contains the individual scores computed by the algorithm for each
 criterion, rated by a user for a given video.
 
-    score:
-
-    TO DESCRIBE. How is it built? What does it influence?
-
-    voting right:
+List of columns:
+
+- public_username:
+  The public username of the user who has included the video in their
+  comparisons
+- video:
+  The ID of the video on YouTube.
+- criteria:
+  The name of the criterion on which the score applies.
+- score:
+  This field is computed by Tournesol's algorithms. The score is a number
+  between -100.0 and +100.0. Individual scores are computed based on all the
+  comparisons submitted by a user. Additionally individual scores of all users
+  are globally scaled such that they are meaningfully comparable and can be
+  aggregated.  
+- voting_right:
+  This field is computed by Tournesol's algorithms. The voting right is a
+  number between 0.0 and 1.0 used to compute collective scores. It is assigned
+  to each user depending on their trust scores (see file users.csv) and on how
+  many users have rated the same video. This is done in such a manner that if a
+  large number of users with low trust scores compare the same video, then they
+  will all receive a very small voting right, but if the number of users with
+  low trust score is not more than 10% of all the users that rated the same
+  video then they will receive a full voting right.
+
+collective_criteria_scores.csv
+------------------------------
 
-    TO DESCRIBE. How is it built? What does it influence?
+This file contains the collective scores computed by the algorithm for each
+criterion.
+
+List of columns:
+
+- video:
+  The ID of the video on YouTube.
+- criteria:
+  The name of the criterion on which the score applies
+- score:
+  This field is computed by Tournesol's algorithms. This score is obtained by
+  considering all the individual scores on the same video and criteria. With a
+  lot of contributors this scores should be close to the median of their
+  individual scores. With a small number of contributors, this score would be
+  reduced towards 0 such that it is not possible for a small number of users to
+  give a very high collective score on a video.
+- publication_date:
+  The date at which the video was uploaded
+- views:
+  Number of views on the video. This field is only updated at the time of the
+  last comparison on the video, so the real number of views will be higher and
+  can be obtained from YouTube API.
+- name:
+  The title of the video.
+- uploader:
+  The name of the Channel that uploaded the video
```

Comment:
As these columns are not present in the dataset anymore, I propose to delete them.

```suggestion
```